redaxo/src/addons/backup/lib/backup.php

```diff
@@ -10,9 +10,6 @@ class rex_backup
     public const IMPORT_EVENT_PRE = 3;
     public const IMPORT_EVENT_POST = 4;
 
-    /**
-     * @return string
-     */
```

Comment:
Hmm, hat das der CS Fixer entfernt?
Falls ja, verstehe ich nicht wieso. Der sollte eigentlich nur Docs entfernen, wenn sie schon über native Type Hints gesetzt sind.
Genauso bei den anderen Stellen.

Reply:
Ich war's nicht

Reply:
Es scheint mit deinem `remove filter for rex_user` commit reingekommen zu sein, falls ich das richtig sehe.

Reply:
Davor ist ja der Merge-Commit. Da sind diese Zeilen wahrscheinlich überhaupt erst reingekommen, glaube die sind noch recht frisch.
Und irgendwie hast du sie dann mit dem nächsten Commit wieder entfernt.

Wie auch immer: Du müsstest diese gelöschten Docs wieder hinzufügen.

Reply:
ok, bewusst habe ich da nichts gemacht. Ich hätte mir vermutlich erst den aktuellen Stand ziehen müssen von redaxo/master, richtig?

Reply:
Ah, mist, ich sehe, das sind noch viel mehr Stellen als nur die eine...